cli/unipipe/commands/generate.ts

```diff
@@ -129,7 +126,7 @@ async function generateTransformHandler(options: CatalogOpts) {
         outputContent = basicTransformHandler.replaceAll('$SERVICEID', options.uuid )
       }
       console.log(outputContent);
-      writeDirectoryWithUserInput("handler.js", outputContent, "Pick a destination directory for the generated transform file:", (options.destination?options.destination:undefined))
+      writeDirectoryWithUserInput("handlers.js", outputContent, "Pick a destination directory for the generated transform file:", (options.destination?options.destination:undefined))
```

Comment:
i have advised that separating each handler.js file for each OSB service. so my idea is every service will have their own unique folder which includes its handler.js and execution-script.sh. so that it wont be so complicated to understand these files. what do you think about it? if we agree on this, i guess `handlers.js` could be a wrong naming choice

Reply:
I agree that structuring via folders and moving handler definition inside folders they manage is a good way of structuring a UniPipe repository. At the same time the format of the definition file allows a registry of handlers, not just a single handler, so making it clear in the name that the file can hold several handlers is a good approach in my opinion.